src/main/resources/static/data-mapping.js

```diff
@@ -9,20 +9,23 @@ var mapped_ocitems = [];//an array containing the paths of mapped oc items
 var baseSvg;
 var treeg, listg;
 
-var tipDiv;
+var tipDiv;//the tip used for items in OC tree
```

Comment:
just a small styling tip: if you press Alt+Cmd+ L in intelliJ on Mac (assuming default settings) it will nicely style your source code, including spaces between comments and the code